plugin/adsk/plugin/importTranslator.cpp

```diff
@@ -114,12 +125,15 @@ UsdMayaImportTranslator::reader(
             /* importWithProxyShapes = */ false,
             timeInterval);
 
-    MayaUsd::ImportData importData(fileName);
-    UsdMaya_ReadJob mUsdReadJob(
-        (fileName != MayaUsd::ImportData::cinstance().filename()) ? importData : MayaUsd::ImportData::cinstance(),
-        jobArgs);
+    UsdMaya_ReadJob mUsdReadJob(importData, jobArgs);
     std::vector<MDagPath> addedDagPaths;
     bool success = mUsdReadJob.Read(&addedDagPaths);
+
+    // After a successful import we clear the import data as we don't want to
+    // re-use it on a subsequent import.
+    if (success)
```

Comment:
New behavior from design. This will fix the problem where a user imports from dialog and then runs manual command. Since the import data will be cleared out it won't be re-used for next manual command.